wp-resources/plugins/sil-dictionary-webonary/webonary/Webonary_Cloud.php

```diff
@@ -198,15 +216,19 @@ public static function entryToReversal($entry): stdClass {
         return $reversal;
     }
 
-    public static function getBlogDictionaryId() {
+    public static function getBlogDictionaryId(): string {
         if (function_exists('is_subdomain_install')) {
             return (
                 is_subdomain_install()
                 ? explode('.', $_SERVER['HTTP_HOST'])[0]
                 : str_replace('/', '', get_blog_details()->path)
             );
         }
-        return WEBONARY_CLOUD_DEFAULT_DICTIONARY_ID;
+
+		if (defined('WEBONARY_CLOUD_DEFAULT_DICTIONARY_ID'))
```

Comment:
nit: Can you fix the indention here so it doesn't look like there's two returns on the same level?

Reply:
@johntseng done. I reformatted the file so it now corresponds to the settings in `.editorconfig`.